assets/scss/app.scss

```diff
@@ -1,4 +1,4 @@
-@import "$$theme";
+@import "hof/frontend/themes/gov-uk/styles/govuk";
```

Comment:
leaner question, where is the path for this?
Is this pointing to `hof-theme-govuk`? and if yes how does this work? so lost